### ISSUE TITLE
openvpn broken with latest release

### ISSUE THREAD

**Issue by hdlineage** [binhex/arch-qbittorrentvpn#149]
updated to latest release, the container ran into issues during start up.
I'm using openvpn with keepsolid vpn. I checked the log and it kept saying "OPENSSL error: ca md too weak."

Reverted to previous release and everything starts fine again.

**Comment by 0x09AF** [comment]
Is there an update to this?
I have tried the latest container 2 times and no luck. Different forums suggest editing _/etc/openssl/config_ - I edited it in the container and on the host (lxc Ubuntu), still with no luck (I enabled legacy config). As soon as I pull the older container (v4.4.5) everything starts working again.

**Comment by droopified** [comment]
sorry to hijack but could you reply with a quick how to with steps on setting up keepsolid with this container? i generated the openvpn file at keepsolid and entered the generated info but cant connect.

**Comment by 0x09AF** [comment]
@droopified What is keepsolid? I assume it's your VPN provider...
If so, check if this provider is in arch-qbittorrentvpn list of supported providers. If no, it requires a little bit of manual configuration.
Had your config been working prior to the OpenSSL upgrade to v3? If so, just add the edits in my message above

**Comment by droopified** [comment]
> What is keepsolid? I assume it's your VPN provider...
yes, the op mentioned keepsolid vpn unlimited as the vpn provider so was wondering what are the steps to get it to connect. i cant for the life of me find a list of supported providers which would be very useful to me since i need a new vpn. i was on private internet access for the last like 10 years but it expired just last week.

**Comment by binhex** [comment]
There is no list because this image will support any VPN provider that can generate a valid OpenVPN or wireguard configuration file
